src/Microsoft.AspNetCore.SignalR/SignalRDependencyInjectionExtensions.cs

```diff
@@ -8,10 +8,11 @@
 
 namespace Microsoft.Extensions.DependencyInjection
 {
-    public static class DependencyInjectionExtensions
+    public static class SignalRDependencyInjectionExtensions
     {
         public static ISignalRBuilder AddSignalR(this IServiceCollection services)
```

Comment:
While you're here should we make this more like other projects and not have an `ISignalRBuilder` and make an overload that accepts `Action<SignalROptions>`